### PR TITLE
Added softlayer_network_vlan service

### PR DESCRIPTION
**Pull request by jianqiu**
Added softlayer_network_vlan service

### CODE REVIEW

**Comment by jianqiu** [comment]
@maximilien 

It is ready for your review, thanks!

cc: @mattcui , @zhanggbj

File: services/softlayer_network_vlan.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		client: client,
+	}
+}
+
+func (slhs *softLayer_Network_Vlan_Service) GetName() string {
+	return "SoftLayer_Network_Vlan"
+}
+
+func (slhs *softLayer_Network_Vlan_Service) GetObject(id int) (datatypes.SoftLayer_Network_Vlan, error) {
+
```
Comment (maximilien) [inline services/softlayer_network_vlan.go]: remove new line

File: services/softlayer_network_vlan_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			fakeClient.FakeHttpClient.DoRawHttpRequestResponse, err = testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Network_Vlan_Service_getObject.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("sucessfully retrieves SoftLayer_Network_Vlan object", func() {
+			networkVlan, err := networkVlanService.GetObject(123456)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(networkVlan.Id).To(Equal(1234567))
+			Expect(networkVlan.NetworkSpace).To(Equal("PUBLIC"))
+		})
```
Comment (maximilien) [inline services/softlayer_network_vlan_test.go]: Please add checks for every field in the struct, so all of these from your fixture:

```
 "accountId": 123456,
 "id": 1234567,
 "modifyDate": "2014-07-28T09:46:29+08:00",
 "name": "Env4 LON02 Fab Pub",
 "primarySubnetId": 123456,
 "vlanNumber": 887,
 "networkSpace": "PUBLIC
```

File: data_types/softlayer_network_subnet.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	"fmt"
+	"net"
+)
+
+type SoftLayer_Network_Subnet struct {
+	Id                int    `json:"id"`
+	NetworkIdentifier string `json:"networkIdentifier"`
+	Gateway           string `json:"gateway"`
+	BroadcastAddress  string `json:"broadcastAddress"`
+	Netmask           string `json:"netmask"`
```
Comment (jianqiu) [inline data_types/softlayer_network_subnet.go]: Exactly, all these fields will be retrieved from SL response. Thanks!

**Comment by jianqiu** [comment]
@maximilien 
I just addressed all comments, could you please review them again? Thanks!

**Pull request merged**